groups/utils.js

```diff
@@ -76,11 +77,25 @@ async function addGroupRoleToMember(memberRoleBody) {
       body: JSON.stringify(memberRoleBody),
     });
 
+    if (!res.ok) {
+      const error = await res.json();
+      throw error;
+    }
+
     const data = await res.json();
     return data;
   } catch (err) {
-    return err;
+    throw err;
+  }
+}
+
+function removeGroupKeywordFromDiscordRoleName(groupName) {
+  if (/^group-.*/.test(groupName)) {
```

Comment:
Should the regex be `group` instead of `group-`?
```suggestion
  if (/^group.*/.test(groupName)) {
```

Reply:
Yes, logically we can do with `^group.*`.
I just added a hyphen for cases like `groupTwitter-developers` (this kind of group is not possible yet).

However, since it's not possible for now, there is no point in adding the extra hyphen. I will remove it.

Thanks for the suggestion.

Reply:
fixed.